packages/story-editor/src/components/checklist/karma/checklist.karma.js

```diff
@@ -82,6 +82,10 @@ describe('Checklist integration', () => {
           type: 'image',
           mimeType: 'image/jpg',
           src: 'http://localhost:9876/__static__/earth.jpg',
+          alt: 'earth',
```

Comment:
so i'm pretty sure that this is causing the test failure for `should open the design inspector panel and focus the text input` because we're relying on having that alt value as missing to generate accessibility issues.

Reply:
Hmm then it should probably be an empty string instead of `undefined`. I'll give it a try.